src/js/components/DataTable/Body.js

```diff
@@ -234,13 +234,20 @@ const Body = forwardRef(
             : undefined
         }
         // The WCAG recommendation for checkboxes is to select them with "Space"
-        onSpace={() => {
-          if (clickableRow) {
-            if (onClickRow === 'select') {
-              selectRow();
-            }
-          }
-        }}
+        onSpace={
```

Comment:
Can we prevent the page from scrolling when we interact with a row via the 'space' key?

Reply:
@jcfilben Can you please screen recording of that behaviour?

Reply:
Sure, here is a video

https://github.com/user-attachments/assets/d2d9ec3a-dd79-4c74-83d0-cc06024578e0

Reply:
Done